admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.js

```diff
@@ -113,9 +114,17 @@ export default {
   orderTaxesTotal: '#orderTaxesTotal',
   orderTotal: '#orderTotal',
   orderHookTabsContainer: '#order_hook_tabs',
-  displayPartialRefundBtn: 'button.partial-refund-display',
-  cancelPartialRefundBtn: 'button.partial-refund-cancel',
   actionColumnElements: 'td.cellProductActions, th.product_actions',
-  togglePartialRefundForm: '.partial-refund:not(.hidden), .shipping-price, .refund-checkboxes-container',
-  productAddNewInvoiceInfo: '#addProductNewInvoiceInfo',
+  // Product cancel/refund elements
+  togglePartialRefundForm: '.cancel-product-element:not(.hidden):not(.shipping-refund), .cancel-product-amount',
+  toggleStandardRefundForm: '.cancel-product-element:not(.hidden):not(.shipping-refund), .cancel-product-selector',
+  displayPartialRefundBtn: 'button.partial-refund-display',
+  displayStandardRefundBtn: 'button.standard-refund-display',
+  cancelProductForm: 'form[name="cancel_product"]',
```

Comment:
```suggestion
  cancelProduct: {
    form: 'form[name="cancel_product"]',
    ...
  }
```

Adding a depth level is easier to read.

Reply:
Done